scylla/src/transport/session.rs

```diff
@@ -46,7 +46,7 @@ use openssl::ssl::SslContext;
 
 /// `Session` manages connections to the cluster and allows to perform queries
 pub struct Session {
-    cluster: Cluster,
+    pub cluster: Cluster,
```

Comment:
Exposing `cluster` as public will cause it to become part of public API, which is undesirable. If your goal was to provide a view to `keyspaces` field of `ClusterData`, It would be better to make a getter in form of public `Session` member function.